ansible-doc.el

```diff
@@ -159,6 +159,9 @@ buffer-local wherever it is set."
          (default (if (or (null modules) (member symbol modules))
                       symbol
                     nil))
+         (prompt (if default
+                     (concat prompt " [" default "]: ")
```

Comment:
Please use ` (default …): ` for the prompt, for consistency with other similar prompts in Emacs (e.g. `describe-function`).